guides/common/modules/proc_upgrading-the-external-database.adoc

```diff
@@ -1,25 +1,18 @@
-[id="Upgrading_the_External_Database_{context}"]
+[id="Upgrading_the_External_Database{context}"]
 = Upgrading the external database
 
-You can upgrade an external database from {EL} 8 to {EL} 9 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
+The external database will be upgraded when running the {foreman-installer} on the {Project} server.
 
 .Prerequisites
-* Create a new {EL} 9 based host for PostgreSQL server that follows the external database on {EL} 9 documentation.
-ifdef::katello,orcharhino,satellite[]
-For more information, see {InstallingServerDocURL}using-external-databases_{project-context}[Using External Databases with {Project}].
-endif::[]
-* Install PostgreSQL version 13 on the new {EL} host.
+* PostgreSQL version 13 is installed on the {EL} host.
+* The {InstallingServerDocURL}using-external-databases_{project-context}[Using External Databases with {Project}] guide has been followed.
```

Comment:
```suggestion
```
I don't think you need this prerequisite. The documentation you're referring to covers configuring a server with an external database. But I think it's clear that any user who wants to upgrade an external database has previously configured such a database. (My suggestion on line 4 also covers that.)